controllers/withwatcher/watcher_controller_test.go

```diff
@@ -48,37 +48,26 @@ func allCRsDeleted() func(customIstioClient *istio.Client) {
 			Expect(controlPlaneClient.Delete(suiteCtx, watcherCr)).To(Succeed())
 		}
 		// verify
-		Eventually(isCrDeletionFinished(), Timeout, Interval).Should(BeTrue())
+		Eventually(isWatcherCrDeletionFinished(), Timeout, Interval).Should(BeTrue())
 		Eventually(isVsRemoved(suiteCtx, customIstioClient)).Should(BeTrue())
 	}
 }
 
-var _ = Describe("Watcher CR scenarios", Ordered, func() {
+var _ = Describe("Watcher CR scenarios", Ordered, Serial, func() {
```

Comment:
Ordered - I thought we decided to get rid of them?
Is setting Serial by mistake?